Sources/SwiftFormats/FormatStyle+CoreGraphics.swift

```diff
@@ -44,6 +44,32 @@ public struct CGPointParseStrategy: ParseStrategy {
 
 // MARK: -
 
+public extension CGPoint {
+    init(_ string: String) throws {
+        self = try CGPointFormatStyle().parseStrategy.parse(string)
+    }
+
+    init<T, Value>(_ value: Value, format: T) throws where T: ParseableFormatStyle, Value: StringProtocol, T.Strategy == CGPointParseStrategy {
+        self = try format.parseStrategy.parse(value.description)
+    }
+
+    init<T, Value>(_ value: Value, strategy: T) throws where T: ParseStrategy, Value: StringProtocol, T.ParseInput == String, T.ParseOutput == CGPoint {
+        self = try strategy.parse(value.description)
+    }
+}
+
+@available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
+public extension ParseableFormatStyle where Self == CGPointFormatStyle {
+    static var cgPoint: Self { .init() }
```

Comment:
I point 'point' is a better name. There are other point types (e.g. UnitPoint, IntegerPoint in my other codebases) that may benefit from formatters and I think a common where possible name is beneficial.